.github/workflows/team-updates.yaml

```
@@ -0,0 +1,24 @@
+name: Team Updates
+on:
+
+  # enable manual trigger via github actions interface
+  workflow_dispatch:
+
+  # trigger on push to main branch
+  push:
+    branches:
+      - main
+
+jobs:
+  # Send a cross-repository workflow dispatch request to the team-management repo
+  # Team management repo workflow will take action on event_type community_teams_update
+  dispatch-team-management:
+    if: github.repository == 'cilium/community'
+    name: Dispatch Team Management Workflow
+    runs-on: ubuntu-latest
+    environment: team-management  
+    steps:
+      # Send the workflow dispatch event via GitHUB API
+      - name: dispatch workflow request
```

nit that might end up bothering me if I end up looking at the actions tab for this from time to time ;) 

```suggestion
    name: Dispatch Team Management
    runs-on: ubuntu-latest
    environment: team-management  
    steps:
      # Send the workflow dispatch event via GitHUB API
      - name: Dispatch workflow request
```

(Noting that the `name` here is actually the _job_ and not the workflow so it'd show up on the left hand side here: https://github.com/cilium/cilium/actions/runs/12715914362/job/35449088454, but also putting "workflow" into the name of a workflow job is redundant).